torch/distributions/studentT.py

```diff
@@ -76,8 +76,7 @@ def log_prob(self, value):
         if self._validate_args:
             self._validate_sample(value)
         y = (value - self.loc) / self.scale
-        Z = (self.scale.log() +
-             0.5 * self.df.log() +
+        Z = (0.5 * self.df.log() +
```

Comment:
This change is incorrect. You need to rescale the function too under change of variable, and this accounts for it.

Also, density functions can have values greater than 1.